File: java/src/com/stockmarket/models/Trade.java
package com.stockmarket.models;

public class Trade {
    private Integer buyOrderId;
    private Integer sellOrderId;
    private Integer buyPrice;
    private Integer sellPrice;
    private Integer quantity;
    private Integer id;
    private String bookName;

    public Trade(Integer buyOrderId, Integer sellOrderId, Integer buyPrice, Integer sellPrice, Integer quantity, Integer id, String bookName) {
        this.buyOrderId = buyOrderId;
        this.sellOrderId = sellOrderId;
        this.buyPrice = buyPrice;
        this.sellPrice = sellPrice;
        this.quantity = quantity;
        this.id = id;
        this.bookName = bookName;
    }

    public Integer getId() {
        return id;
    }

    public String getBookName() {
        return bookName;
    }

    public Integer getQuantity() {
        return quantity;
    }

    public Integer getSellPrice() {
        return sellPrice;
    }

    public Integer getBuyPrice() {
        return buyPrice;
    }

    public Integer getSellOrderId() {
        return sellOrderId;
    }

    public Integer getBuyOrderId() {
        return buyOrderId;
    }
}

File: kotlin/src/com/stockmarket/models/Trade.kt
package com.stockmarket.models

class Trade(buyOrderId: Int, sellOrderId: Int, buyPrice: Int, sellPrice: Int, quantity: Int, id: Int, bookName: String) {
    private val buyOrderId: Int
    private val sellOrderId: Int
    private val buyPrice: Int
    private val sellPrice: Int
    private val quantity: Int
    private val id: Int
    val bookName: String
    fun getId(): Int {
        return id
    }

    fun getQuantity(): Int {
        return quantity
    }

    fun getSellPrice(): Int {
        return sellPrice
    }

    fun getBuyPrice(): Int {
        return buyPrice
    }

    fun getSellOrderId(): Int {
        return sellOrderId
    }

    fun getBuyOrderId(): Int {
        return buyOrderId
    }

    init {
        this.buyOrderId = buyOrderId
        this.sellOrderId = sellOrderId
        this.buyPrice = buyPrice
        this.sellPrice = sellPrice
        this.quantity = quantity
        this.id = id
        this.bookName = bookName
    }
}